examples/quickstart/jupyter-notebooks/README.md

```diff
@@ -65,8 +65,15 @@ Make sure you meet the following requirements before starting the Jupyter-based
     jupyter notebook --port 3001
     ```
 
-- An available Druid instance. You can use the `micro-quickstart` configuration
-  described in [Quickstart](https://druid.apache.org/docs/latest/tutorials/index.html).
+- The Python API client for Druid. Clone the Druid repo if you haven't already.
+Go to your Druid source repo and install `druidapi` with the following commands:
+
+  ```bash
+  cd examples/quickstart/jupyter-notebooks/druidapi
+  pip install .
+  ```
+
+- An available Druid instance. You can use the [quickstart deployment](https://druid.apache.org/docs/latest/tutorials/index.html).
```

Comment:
maybe not for this PR, but if we're having them start Juypter in a notebook, we can also give them the option to fire up Druid that way.